src/views/ApiKeyView.py

```diff
@@ -0,0 +1,42 @@
+from flask import request, json, Response, Blueprint, jsonify
+from marshmallow import ValidationError
+from sqlalchemy import exc
+from ..models.ApiKey import ApiKey, ApiKeySchema
+from . import custom_response, api_key_check
+import secrets
```

Comment:
I'd explain this, but its a secret

Reply:
just a quick way to generate secure random strings

Reply:
The secret sauce